src/mmw/js/src/modeling/views.js

```diff
@@ -66,6 +67,17 @@ var ModelingHeaderView = Marionette.LayoutView.extend({
 
     onShow: function() {
         this.reRender();
+    },
+
+    saveAfterLogin: function(user, guest) {
+        if (!guest && this.model.isNew()) {
+            var user_id = user.get('id');
+            this.model.set('user_id', user_id);
```

Comment:
The `user_id` seems to be required as of #348 to correctly save projects and scenarios, otherwise they are not associated with each other. But I agree, either we are overusing the `user_id` field here or we are not handling it well enough on the server side. I didn't give it much thought at the time, but will investigate this tomorrow morning.